pkg/templates/resources/resources.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2021 The KubeOne Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+const (
+	VirtualIP = "169.254.20.10"
```

Comment:
```suggestion
	NodeLocalDNSVirtualIP = "169.254.20.10"
```

Reply:
Previously package name was part of the identity, but now since it's "no more", we need to incorporate it into the const name.